lms/templates/registration/password_reset_complete.html

```diff
@@ -6,49 +6,24 @@
 {% endblock %}
 
 {% block bodyextra %}
-  <script type="text/javascript">
-    $(function() {
-
-      // adding js class for styling with accessibility in mind
-      $('body').addClass('js');
-
-      // new window/tab opening
-      $('a[rel="external"], a[class="new-vp"]')
-      .click( function() {
-      window.open( $(this).attr('href') );
-      return false;
-      });
-
-      // form field label styling on focus
-      $("form :input").focus(function() {
-        $("label[for='" + this.id + "']").parent().addClass("is-focused");
-      }).blur(function() {
-        $("label").parent().removeClass("is-focused");
-      });
-    });
```

Comment:
This code has been moved to ```password_reset.js``` so is still being executed on this page, however because it is the reset complete page I'm not sure that there are any form elements present. Can you please confirm which form elements are present on this page?